txstratum/protocol.py

```diff
@@ -68,6 +69,13 @@ class StratumProtocol(JSONRPCProtocol):
     MESSAGES_TIMEOUT_INTERVAL = (
         5  # in seconds, the interval to clear timed out messages
     )
+    RTT_ESTIMATOR_INTERVAL = (
+        5  # in seconds, to estimate the RTT from application to miner
+    )
+    RTT_ESTIMATOR_MESSAGE_TIMEOUT = (
+        60  # in seconds, the maximum time to wait for an rtt message to be received
+    )
+    RTT_CONFIDENCE_CONSTANT = 0.7
```

Comment:
I'd call it `RTT_ALPHA_SLIDING_WINDOW`

Reply:
Done https://github.com/HathorNetwork/tx-mining-service/pull/114/commits/a8ffaedd2d48fe5a75a0a22ae1322d2ebbff82ca